app/api/routes_admin.py

```
@@ -0,0 +1,710 @@
+                    for name in dirs:
+                        try:
+                            os.rmdir(os.path.join(root, name))
+                        except Exception:
+                            pass
+                try:
+                    os.rmdir(p)
+                except Exception:
+                    pass
+                deleted.append(p)
+            # Also remove corresponding archive if present
+            arch = os.path.join(base, "archives", f"{model_id}-{os.path.basename(p)}.zip")
```

Ruff (0.13.1)

Undefined name `base`

(F821)